jobs/registry-change.js

```diff
@@ -64,18 +65,14 @@ module.exports = async function (
   }
   await npm.put(updatedAt(Object.assign(npmDbDoc, npmDoc)))
 
-  // currently we only handle latest versions
-  // so we can heavily optimise by exiting here
-  // we want to handle different distTags in the future
-  if (distTag !== 'latest') {
-    log.info(`exited: ${dependency} distTag is ${distTag} (not latest)`)
+  const version = distTags[distTag]
+  if (semver.prerelease(version) && distTag === 'latest') {
```

Comment:
Ah, what about stuff like https://www.npmjs.com/package/@babel/core where everything is on latest?